smartsim/settings/launchSettings.py

```diff
@@ -67,7 +67,7 @@ def launcher(self) -> str:
         return self._launcher.value
 
     @property
-    def launch_args(self) -> LaunchArgBuilder:
+    def launch_args(self) -> LaunchArgBuilder[t.Any]:
         """Return the launch argument translator."""
         return self._arg_builder
 
```

Comment:
That seems reasonable to me! The setter here seemed like a nice short-hand, but is definitely not something I am _that_ tied to, and could VERY easily be convinced it is more confusing than helpful.

Since we don't want "average" users to be manually constructing `LaunchArgBuilder`s themselves, thoughts on just removing the setter entirely? Or should we keep it around for "power" users just in case with a change of input type??

Reply:
Decided to removed the `@launch_args.setter` entirely.